pyslurm/pyslurm.pyx

```diff
@@ -5467,30 +5467,30 @@ cdef class slurmdb_jobs:
                 JOBS_info[u'show_full'] = job.show_full
                 JOBS_info[u'start'] = job.start
                 JOBS_info[u'state'] = job.state
-                JOBS_info[u'state_str'] = slurm.slurm_job_state_string(job.state)
+                JOBS_info[u'state_str'] = slurm.stringOrNone(slurm.slurm_job_state_string(job.state), '')
```

Comment:
Does `slurm_job_state_string` ever return `null`?

Reply:
It returns byte array, e.g. b'123' which breaks things like json.dumps.